### ISSUE TITLE
A14 Support

### ISSUE THREAD

**Issue by JustW0LF** [X1nto/WeakAuth#3]
Thanks for the mod, it runs well on my android 13 device. But on my android 14 device (crdroid) it doesn't appear to do anything. Same installation method, magisk + Lsposed zygisk. I'm forced to enter my pin after 5 wrong fingerprints. I haven't gotten to the 72 hours mark yet. Does this work at all for A14?

**Comment by X1nto** [comment]
Haven't tested for Android 14. I'll check it out

**Comment by amosbird** [comment]
> Haven't tested for Android 14. I'll check it out

Same here. Have you got around to checking out the later Android versions?

**Comment by X1nto** [comment]
I made a new release that should fix that issue. I don't have an Android 14 device to test on, so please report back if it doesn't fix issues.

**Comment by JaxonJJB** [comment]
It doesn't seem to be working from my testing on reboot or many failed fingerprint unlocks. Would you like the LSPosed module logs?

Also you're amazing thank you

**Comment by X1nto** [comment]
Logs would be helpful, there's a chance I hooked something wrong.

**Comment by JaxonJJB** [comment]
[verbose-LSPosed_2025-08-24T09_54_04.139436.zip](https://github.com/user-attachments/files/21957535/verbose-LSPosed_2025-08-24T09_54_04.139436.zip)

[modules-LSPosed_2025-08-24T09_52_50.325423.zip](https://github.com/user-attachments/files/21957536/modules-LSPosed_2025-08-24T09_52_50.325423.zip)

**Comment by X1nto** [comment]
Found the issue.
<img width="1068" height="437" alt="Image" src="https://github.com/user-attachments/assets/4f7a2955-d68e-41bc-8678-a2f129066897" />

**Comment by X1nto** [comment]
@JaxonJJB what's your device?

**Comment by JaxonJJB** [comment]
Im on a pixel 7a (running android 14 build number AP2A.240905.003, if thats important) . I'm not the most familiar with this environment, but it looks like a problem with zygote/zygisk?

**Comment by X1nto** [comment]
No, I'm probably hooking in the wrong place. The very first approach that worked before hooked in the Zygote, but I guess that's too early for the KeyGuard. I'm sending another APK here (inside a ZIP because GitHub doesn't allow APKs). If it works, I'll release it to GitHub Releases.

[app-release.zip](https://github.com/user-attachments/files/21957756/app-release.zip)

**Comment by JaxonJJB** [comment]
I tried it, didn't seem to work under the same tests. Heres the logs. Also, don't stress about trying to fix this here and now, I'm just tinkering and this is not time sensitive whatsoever

[verbose-LSPosed_2025-08-24T10_55_31.337585.zip](https://github.com/user-attachments/files/21957821/verbose-LSPosed_2025-08-24T10_55_31.337585.zip)

[modules-LSPosed_2025-08-24T10_55_16.146485.zip](https://github.com/user-attachments/files/21957822/modules-LSPosed_2025-08-24T10_55_16.146485.zip)

**Comment by X1nto** [comment]
Same issue, I'll try to spin up an Android 14 emulator with Xposed if I can find one. Thanks for the logs.

**Comment by JaxonJJB** [comment]
No problem! Thanks for the help

**Comment by X1nto** [comment]
@JaxonJJB Could you test if this APK works?

[app-release.zip](https://github.com/user-attachments/files/22541134/app-release.zip)

**Comment by JaxonJJB** [comment]
Just tried it, no dice :/ hopefully these help

[LSPosed_2025-09-25T12_27_08.005799.zip](https://github.com/user-attachments/files/22542662/LSPosed_2025-09-25T12_27_08.005799.zip)

[LSPosed(verbose)_2025-09-25T12_27_20.492522.zip](https://github.com/user-attachments/files/22542664/LSPosed.verbose._2025-09-25T12_27_20.492522.zip)

**Comment by X1nto** [comment]
I should've added better logging. How's this one?

[app-release.zip](https://github.com/user-attachments/files/22545389/app-release.zip)

**Comment by JaxonJJB** [comment]
Unfortunately my phone force updated itself to android 16 so I cant test it anymore

**Comment by luminous030** [comment]
> I should've added better logging. How's this one?
> 
> [app-release.zip](https://github.com/user-attachments/files/22545389/app-release.zip)

Last release package not working for me.
I will using my OnePlus 15 to test this version.

**Comment by luminous030** [comment]
- install & enable module
- reboot my phone
- still need password

[1.zip](https://github.com/user-attachments/files/25535031/1.zip)
[2.zip](https://github.com/user-attachments/files/25535032/2.zip)
